modules/bbp-simulation/src/test/resources/data/hbphackaton/simulation/tracefeatureextraction/v0.1.0/min-fields-ignored.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
was this file moved or deleted?

Reply:
Moved at: data/bbp/simulation/tracefeatureextraction/v0.1.0/min-fields-ignored.json
Part of this pull request as well.

Reply:
Missed that